lib/fetch/index.js

```diff
@@ -1361,6 +1361,7 @@ function httpRedirectFetch (fetchParams, response) {
 
   // 18. Append locationURL to request’s URL list.
   request.urlList.push(locationURL)
+  // TODO: not enough?
```

Comment:
Bug?

After redirecting, you dont have the url of the request object. Does it have to be in the response? Does reponse even have a url field?

This does not change request.url, which is probably connected with bug report #2520

request.url is using a getter:

https://github.com/nodejs/undici/blob/3372aed902951f135d710d0fb430f39b8571e547/lib/fetch/request.js#L577

So either urlList gets a Proxy to set the corresponding request[kState].url or url is retrieved dynamically by taking the last element of request[kState].urlList

Reply:
```suggestion
```